### PR TITLE
Update Style-guide.md [skip ci]

### PR DESCRIPTION
**Pull request by michaelbadcrumble**
Added a helpful guideline for naming variables in Meson.

### CODE REVIEW

File: docs/markdown/Style-guide.md [hunk 1]
```diff
@@ -11,6 +11,12 @@ Meson build files.
 
 Always spaces.
 
+## Nameing Variable
```
Comment (TheQwertiest) [inline docs/markdown/Style-guide.md:14]:
```suggestion
## Naming Variable
```

**Comment by michaelbadcrumble** [comment]
Ping

File: docs/markdown/Style-guide.md (hunk 1 above)
Comment (michaelbadcrumble) [reply docs/markdown/Style-guide.md]: Ok fixed the typo.

File: docs/markdown/Style-guide.md [hunk 2]
```diff
@@ -11,6 +11,12 @@ Meson build files.
 
 Always spaces.
 
+## Naming Variable
+
+The most consistent naming convention is the snack case. Let say you would
```
Comment (jpakkane) [inline docs/markdown/Style-guide.md:16]: This should probably be "snake case" rather than "snack case".

Comment (michaelbadcrumble) [reply docs/markdown/Style-guide.md]: Ok typo is fixed.

**Pull request merged**